src/index.js

```diff
@@ -144,7 +145,20 @@ export class WT {
       delete this.loaderImage.onload;
       reject();
     };
-    this.loaderImage.src = `${this.getUrl()}?${query}`;
+
+    fetch(`${this.getRoot()}/wt/t`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
can we switch this to json? suppose can wait until the follow-up release

Reply:
Yeah good question. I initially thought json wouldn't work because I was getting a CORS error, but the change in platform seems to have addressed it now. I can switch to json in the follow-up.